### PR TITLE
Convert pkg/termios to use x/sys/unix

### PR DESCRIPTION
**Pull request by tklauser**
Follow up on #429 and convert pkg/termios to use functions, types and constants from golang.org/x/sys/unix. Also convert cmds/pflask to use the updated termios package instead of locally duplicating its functionality.

The last commit fixes some golint warnings in cmds/pflask (already present before this PR), which I encountered while checking my changes.

### CODE REVIEW

File: cmds/pflask/pflask.go [hunk 1]
```diff
@@ -19,6 +19,9 @@ import (
 	"time"
 
 	"unsafe"
+
+	// "github.com/u-root/u-root/pkg/termios"
```
Comment (rminnich) [inline cmds/pflask/pflask.go]: If this works feel free to remove the commented out package.

File: cmds/pflask/pflask.go [hunk 2]
```diff
@@ -373,14 +375,14 @@ func main() {
 	MountAll(*chroot, unprivileged)
 
 	if !unprivileged {
-		copy_nodes(*chroot)
+		copyNodes(*chroot)
 	}
 
-	make_ptmx(*chroot)
+	makePtmx(*chroot)
```
Comment (rminnich) [inline cmds/pflask/pflask.go]: Note that all of these could be changed to use the support code in uroot/root.go. But it's fine for now. This originally came from minix and you can tell how old it is by all the non-standard things like _, etc.

File: cmds/pflask/pflask.go (hunk 1 above)
Comment (tklauser) [reply cmds/pflask/pflask.go]: Ok, will check. I only rudimentarily tested pflask with this PR, so I will check again whether it still works.

File: cmds/pflask/pflask.go (hunk 2 above)
Comment (tklauser) [reply cmds/pflask/pflask.go]: Good to know, thanks. Will send a follow-up PR to use the code from `uroot/root.go`

**Pull request merged**